### PR TITLE
1579/feat/disable existing event contacts

### PR DESCRIPTION
**Pull request by y-golde**
disabled existing contacts on contact bank and on create/edit contact tabs

also added disabling existing contacts on groupedInvestigationForm

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/GroupedInvestigations/GroupedInvestigationForm/ContactsForm/InvestigationAccordion/AccordionContent/AccordionContent.tsx [hunk 1]
```diff
@@ -11,8 +11,8 @@ import SelectedRowsMessage from './SelectedRowsMessage/SelectedRowsMessage';
 const AccordionContent = (props: Props) => {
     const { events, isGroupReasonFamily, duplicateIds } = props;
     const [query, setQuery] = useState<string>('');
-    const { getCurrentSelectedRowsLength , filteredEvents } = useAccordionContent({events , query});   
-
+    const { getCurrentSelectedRowsLength, filteredEvents, existingIds } = useAccordionContent({events , query});   
+    console.log(existingIds);
```
Comment (Noam6421) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/GroupedInvestigations/GroupedInvestigationForm/ContactsForm/InvestigationAccordion/AccordionContent/AccordionContent.tsx:15]: delete

**Review by Noam6421 — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ContactsBank/ContactsBankTable/Row/Row.tsx [hunk 2]
```diff
@@ -4,27 +4,37 @@ import { TableCell, TableRow, Checkbox, Select, MenuItem, TextField } from '@mat
...
+        handleExtraInfoChange,
+        doesPersonExistInEvent 
     } = UseRow({contact});
 
-    const { personInfo } = contact;
 
+    const isChecked = isPersonChecked();
+    const doesExist = doesPersonExistInEvent();
+
+    const rowClass = doesExist
```
Comment (barnahum8) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ContactsBank/ContactsBankTable/Row/Row.tsx:25]: a little bit confusing, maybe use brackets

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/GroupedInvestigations/GroupedInvestigationForm/ContactsForm/InvestigationAccordion/AccordionContent/AccordionContent.tsx (hunk 1 above)
Comment (barnahum8) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/GroupedInvestigations/GroupedInvestigationForm/ContactsForm/InvestigationAccordion/AccordionContent/AccordionContent.tsx:15]: remove

Comment (y-golde) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/GroupedInvestigations/GroupedInvestigationForm/ContactsForm/InvestigationAccordion/AccordionContent/AccordionContent.tsx]: fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1361/commits/1957a1216bbdac06a1c77391fc36c14e84ae1ecc)

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ContactsBank/ContactsBankTable/Row/Row.tsx (hunk 2 above)
Comment (y-golde) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ContactsBank/ContactsBankTable/Row/Row.tsx]: fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1361/commits/1957a1216bbdac06a1c77391fc36c14e84ae1ecc)

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/GroupedInvestigations/GroupedInvestigationForm/ContactsForm/InvestigationAccordion/AccordionContent/AccordionContent.tsx (hunk 1 above)
Comment (y-golde) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/GroupedInvestigations/GroupedInvestigationForm/ContactsForm/InvestigationAccordion/AccordionContent/AccordionContent.tsx]: fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1361/commits/1957a1216bbdac06a1c77391fc36c14e84ae1ecc)

**Review by Noam6421 — APPROVED** [review]

**Review by barnahum8 — APPROVED** [review]

**Pull request merged**